src/lightbox/LightboxFooter.js

```diff
@@ -36,10 +37,17 @@ export default class LightboxFooter extends PureComponent<Props> {
   render() {
     const { displayMessage, onOptionsPress, style } = this.props;
     return (
-      <View style={[styles.wrapper, style]}>
+      <SafeAreaView
+        mode="padding"
+        // Why we don't do anything about the unsafe area at the
+        // bottom here: `Lightbox` ensures that the footer clears it
+        // when it slides in.
+        edges={['right', 'left']}
+        style={[styles.wrapper, style]}
+      >
```

Comment:
Ah, so perhaps adding `'bottom'` here would be the way to get the footer's padding to extend through to the bottom?